app/src/main/java/org/oppia/app/player/exploration/ExplorationFragment.kt

```diff
@@ -59,10 +59,10 @@ class ExplorationFragment : InjectableFragment() {
   override fun onAttach(context: Context) {
     super.onAttach(context)
     fragmentComponent.inject(this)
-    val storyTextSize =
+    val readingTextSize =
       arguments!!.getString(STORY_DEFAULT_FONT_SIZE_ARGUMENT_KEY)
-    checkNotNull(storyTextSize) { "ExplorationFragment must be created with a story text size" }
-    fontScaleConfigurationUtil.adjustFontScale(context, storyTextSize)
+    checkNotNull(readingTextSize) { "ExplorationFragment must be created with a story text size" }
```

Comment:
Update the message:

`ExplorationFragment must be created with a reading text size`